### PR TITLE
Create repository.datadog.yml

### PR DESCRIPTION
**Pull request by mimfgg**

### CODE REVIEW

File: repository.datadog.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+schema-version: v1
+kind: mergequeue
+team: ci-interfaces
```
Comment (BenoitZugmeyer) [inline repository.datadog.yml:4]: **❓ question:** I don't see any doc for the `team` option. ‏Does it correspond do the GitHub team like `DataDog/[team]`? Shouldn't the team be `rum-browser`?

File: repository.datadog.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+enable: true
+merge_method: merge
```
Comment (BenoitZugmeyer) [inline repository.datadog.yml]: **💬 suggestion:** ‏we are enforcing 'squash' merges on the main branch:

```suggestion
merge_method: squash
```

Comment (mimfgg) [reply repository.datadog.yml]: yeah, I removed the option already, the default for the repo will be used.

File: repository.datadog.yml (hunk 1 above)
Comment (mimfgg) [reply repository.datadog.yml:4]: good question ... I don't think we use the field for the moment. Let's change to rum-browser

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**